.github/workflows/ci.yml

```diff
@@ -115,8 +81,71 @@ jobs:
               -DCMAKE_COMPILE_WARNING_AS_ERROR=${{matrix.warnings-as-errors}}
             cmake --workflow --preset "${{matrix.config}}"
           fi
+
+      - name: Set capitalized config
+        run: echo "CONFIG_CAMEL=$(echo ${{ matrix.config }} | sed 's/debug/Debug/;s/release/Release/')" >> $GITHUB_ENV
+
+      - name: Upload Slang
+        uses: actions/upload-artifact@v4
+        with:
+          name: test-${{matrix.os}}-${{matrix.platform}}-${{matrix.compiler}}-${{matrix.config}}
+          if-no-files-found: error
+          retention-days: 1
+          path: |
+            build/${{ env.CONFIG_CAMEL }}
+            !**/*.pdb
+            !**/*.ilk
+            !**/*.lib
+            !**/*.exp
+            !**/*.a
+
+      - name: Upload slang-LLVM
+        uses: actions/upload-artifact@v4
+        if: ${{ ! matrix.full-gpu-tests }}
+        with:
+          name: slang-build-${{matrix.os}}-${{matrix.platform}}-${{matrix.compiler}}-${{matrix.config}}
+          # The install directory used in the packaging step
+          path: build/dist-${{matrix.config}}/**/ZIP/slang/*
+
+  test:
+    needs: [load_matrix, build]
+    strategy:
+      fail-fast: false
+      matrix: ${{ fromJson(needs.load_matrix.outputs.matrix_test) }}
+    runs-on: ${{ matrix.runs-on }}
+
+    defaults:
+      run:
+        shell: bash
+
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          submodules: 'recursive'
+          fetch-depth: '0'
+
+      - uses: ./.github/actions/common-setup
+        with:
+          os: ${{matrix.os}}
+          compiler: ${{matrix.compiler}}
+          platform: ${{matrix.platform}}
+          config: ${{matrix.config}}
+
+      - name: Set capitalized config
+        run: echo "CONFIG_CAMEL=$(echo ${{ matrix.config }} | sed 's/debug/Debug/;s/release/Release/')" >> $GITHUB_ENV
```

Comment:
Might be easier to set this from common-setup/action.yml, can just export the already made `Config` variable (although I don't know if windows env is case sensitive?)

Reply:
I was thinking to put it in the `matrix` like the following,
```
include:
  { config: debug, config_camel: Debug }
  { config: Release, config_camel: Release }
```
Then I wouldn't need to spawn a bash only to handle the string data.